src/mudlet.cpp

```diff
@@ -4168,19 +4179,18 @@ int mudlet::scanWordList(QStringList& wl, QHash<QString, unsigned int>& gc)
 Hunhandle* mudlet::prepareProfileDictionary(const QString& hostName, QSet<QString>& wordSet)
 {
     // Need to check that the files exist first:
-    QString dictionaryPathFileName(getMudletPath(mudlet::profileDataItemPath, hostName, qsl("profile.dic")));
-    QString affixPathFileName(getMudletPath(mudlet::profileDataItemPath, hostName, qsl("profile.aff")));
-    QFile dictionary(dictionaryPathFileName);
-    QFile affix(affixPathFileName);
+    QString dictionaryPath(getMudletPath(mudlet::profileDataItemPath, hostName, qsl("profile.dic")));
```

Comment:
:-1: I tend to use "pathFileName" because it tells me that the content is a **string** not a ***file*** and contains BOTH elements (directory AND file in that directory) - changing it to just "path" (or "pathName") loses that information...

Reply:
It leads to some very verbose and uncomfortable to work with code. You don't need to be reminded about the fact every single time you invoke the variable, so how about we just mention this fact at the point of definition?